app/helpers/application_helper.rb

```diff
@@ -1,4 +1,8 @@
 module ApplicationHelper
+  def section_name(name = '')
```

Comment:
Not sure yet if this is the best approach with tailwind css, but we need to do something about code duplication and consistency.